test/csuite/timestamp_abort/main.c

```diff
@@ -1500,22 +1592,25 @@ main(int argc, char *argv[])
     struct stat sb;
     WT_LAZY_FS lazyfs;
     pid_t pid;
-    uint32_t rand_value, timeout;
+    uint32_t iteration, num_iterations, rand_value, timeout;
     int ch, status, ret;
     char buf[PATH_MAX], bucket[512];
     char cwd_start[PATH_MAX]; /* The working directory when we started */
     bool rand_th, rand_time, verify_only;
 
     (void)testutil_set_progname(argv);
+    __wt_stream_set_line_buffer(stdout);
 
     opts = &_opts;
     memset(opts, 0, sizeof(*opts));
 
+    backup_full_interval = 4;
     backup_granularity_kb = 16;
-    backup_verify_quick = true;
+    backup_verify_quick = false;
     backup_verify_throughout = false;
```

Comment:
I don't know if this ticket is the right place, but we should figure out the right values for these variables. In my runs 16k granularity has been overwhelming with multiple iterations. I end up with over 16k bits in the bitmap. I have reproduced with a 1Mb granularity that has "only" several hundred bits instead. And verifying after each incremental backup, etc.

Reply:
I think we can just put a few sensible values here, which you can then update by hand when you are actively debugging something. Here are my thoughts:
- For the granularity, let's just use 1024 KB as you have been using in your testing, as all that's really needed here is to use more bits than the default, and 1024 KB seems to trigger that.
- Quick verification should remain turned off until we implement Keith's idea of comparing only allocated parts of each chunk.
- I'd prefer for the "throughout" verification, which I actually just renamed to `backup_verify_immediately`, to remain `false`, because it takes too long to verify the backups right after they are taken. These backups will be tested after the child gets killed anyways. You can turn it on if it helps you when you are actively debugging these issues.
- I made `backup_full_interval` to be a command-line option.